docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
     volumes:
       - ./docker/nginx/vhost.conf:/etc/nginx/conf.d/default.conf:ro
       - ./public/:/srv/public/:ro
+    ports:
```

Comment:
2 issues with that:

* It does not work with 3.7, but 3.6. My docker engine version is 18.09.5 (18.06.0+ required) but have the following message:  

> Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

* When port definitions are removed, the pages cannot be reached unless I create a `docker-compose.override.yml` file. I am not sure to understand why the default ports cannot be used. Any idea?